docs/user-manual/en/amqp-server-DR.md

```diff
@@ -0,0 +1,9 @@
+# AMQP and Multiple Replicas
+
+After 2.16.0 ActiveMQ Artemis added the ability to capture broker events:
+
+You can 
+
+You can use 
+
+For each of these events, the ActiveMQ 
```

Comment:
need to have details around ensuring ports down etc until you want to activate DR, with risks such as queue features such as Mgroups, exlcusive consumer, max consumers will not be gaurenteed.

Reply:
Yeah.. that's what I will add today.